### PR TITLE
Remove DIR_DELIM in Lua

### PR DESCRIPTION
**Pull request by Rui-Minetest**
"/" is available on all platforms.

### CODE REVIEW

File: builtin/profiler/reporter.lua [hunk 1]
```diff
@@ -15,7 +15,7 @@
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-local DIR_DELIM, LINE_DELIM = DIR_DELIM, "\n"
+local DIR_DELIM, LINE_DELIM = "/", "\n"
```
Comment (Rui-Minetest) [inline builtin/profiler/reporter.lua]: Leave to fit with LINE_DELIM.

**Comment by SmallJoker** [comment]
A same named macro is also used in the engine code. Changing it in Lua only LGTM because lots of mods also use the slash (instead of `DIR_DELIM`) to load or read other files.

**Comment by Rui-Minetest** [comment]
Lua supports "/" for delimiter, but (perhaps) C++ doesn't it.

**Comment by paramat** [comment]
Needs testing.

**Comment by Rui-Minetest** [comment]
Tested.

**Comment by SmallJoker** [comment]
> Tested.

It was meant to test this across the different platforms.

**Comment by t4im** [comment]
> It was meant to test this across the different platforms.

Can confirm that standardized Posix path delimiters work on Posix systems like linux, mac, bsd, android.
So you only need to test that one nasty system that never complies with standards everyone else complies with. :-)

**Comment by Rui-Minetest** [comment]
I'm sorry.
I tested this on Windows and Android.

**Pull request closed without merging**